modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -68,16 +68,38 @@ This file describes all the dependencies required by the package to work properl
 
 The plugin packages your application and creates the deployable JAR file into the target directory within your project's folder. +
 
-==== Import Your Generated JAR to Anypoint Studio.
+=== Create an Application Package to Import into Anypoint Studio
 
-If you want to import your generated JAR into Anypoint Studio, you need to package your application using the `-DattachMuleSources` option.
+To generate a JAR file that can be imported into Anypoint Studio, you need to package your application using the `-DattachMuleSources` parameter to include source files and metadata that Studio requires to re-import the deployable file as an open Mule project into your workspace.
 
+From the command line in your project's folder, run:
 [source,console,linenums]
 ----
-> mvn clean package -DattachMuleSources
+$ mvn clean package -DattachMuleSources
 ----
 
-The `-DattachMuleSources` argument tells the plugin to add a mule-src folder inside the META-INF directory with an exact copy of your application structure at design time.
+The `-DattachMuleSources` parameter tells the plugin to add a `mule-src` folder inside the `META-INF` directory with an exact copy of your application structure at design time. This option also packages project modules and dependencies required to create a functioning Mule deployable archive file that can be deployed into a Mule runtime engine.
+
+=== Create a Lightweight Package
+
+You can skip bundling the actual modules and external dependencies required to run the Mule application in a Mule runtime engine, and create a lightweight package with only the source files and metadata required to import the JAR package back into Anypoint Studio.
+
+From the command line in your project's folder, run:
+[source,console,linenums]
+----
+$ mvn clean package -DlightweightPackage
+----
+
+When this parameter is specified, the plugin creates a lightweight JAR file that does not include any dependencies specified in the Mule application's `pom.xml` file. This JAR file cannot be deployed to a Mule runtime engine, it only offers a way to archive just the Mule application's source files. The result of this Maven parameter is the same as unchecking *Include project modules and dependencies* when exporting the Mule application from Anypoint Studio.
+
+=== Specify Multiple Parameters
+
+You can also combine the parameters together to create a deployable Mule application package that also includes the source files and metadata to import the package back into Anypoint Studio.
```

Comment:
Rephrase this without the 'deployable' word. It may confuse the users that this generated jar can be deployed to a mule instance. These 2 parameters will generated a jar that can be imported to Studio and that doesn't ship all the required dependencies for it to run on a Mule instance.

Reply:
Fixed in 3f6e2f5